plugin/src/main/java/io/jenkins/plugins/casc/TokenReloadAction.java

```diff
@@ -0,0 +1,74 @@
+package io.jenkins.plugins.casc;
+
+import com.google.common.base.Strings;
+import hudson.Extension;
+import hudson.model.UnprotectedRootAction;
+import java.io.IOException;
+import java.util.logging.Logger;
+import javax.annotation.CheckForNull;
+import javax.servlet.http.HttpServletRequest;
+import org.apache.commons.httpclient.HttpStatus;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.interceptor.RequirePOST;
+
+@Extension
+public class TokenReloadAction implements UnprotectedRootAction {
+    public static final Logger LOGGER = Logger.getLogger(TokenReloadAction.class.getName());
+
+    public static final String URL_NAME = "/reload-configuration-as-code";
+    public static final String RELOAD_TOKEN_PROPERTY = "casc.reload.token";
+    public static final String RELOAD_TOKEN_QUERY_PARAMETER = "casc-reload-token";
+
+
+    @CheckForNull
+    @Override
+    public String getIconFileName() {
+        return null;
+    }
+
+    @CheckForNull
+    @Override
+    public String getDisplayName() {
+        return "Reload Configuration as Code";
```

Comment:
It'll be better if you could localization here.

Reply:
I tried adding a Messages.properties file but the Messages class didn't get created on a `mvn install` feel free to send a PR to move existing resources in.